components/systemd/component_output.go

```diff
@@ -188,6 +183,12 @@ func CreateGet(cfg Config) query.GetFunc {
 
 			uptimeSeconds := int64(0)
 			uptimeDescription := "n/a"
+
+			uptime, err := systemd.GetUptime(unit)
+			if err != nil {
```

Comment:
We can consider correcting the same errors that have occurred before, if there are not too many. 
In addition, we need to find out the root cause.

Reply:
Like line 227 in that file, we should change the logger level to error.